### ISSUE TITLE
Add bug report (no space) to the list.

### ISSUE THREAD

**Issue by robdodson** [wooorm/retext-profanities#18]
### Subject of the feature

I recently learned that the phrase "bug report" without a space in the middle can be offensive. [Here's a link to the racial slur](https://www.merriam-webster.com/dictionary/bugre).

Without the space it essentially reads as `<racial-slur>port` or `<racial-slur>town`.

Suggested alternatives would be "snapshot" or any phrasing which makes sure to separate the words: "bug report, bug-report".

Does this seem like something worth adding?

**Comment by wooorm** [comment]
Oh interesting! That term, `bugre`, is a Portuguese word, no?

We do maintain a list of Brazilian Portuguese slurs, and this one isn’t listed, so we can add `bugre` there: https://github.com/words/cuss. Although it seems like it’s also the name for a region in Brazil?

I do wonder about `<racial-slur-in-other-language>port`. A different example is [Google’s own pik project](https://github.com/google/pik/issues/6). And what about English words that include English stuff? such as [s*cunt*thorpe](https://en.wikipedia.org/wiki/Scunthorpe_problem)?

The divide between the two projects, profanities and equality, is that the former is more of an unambiguous list of offensive terms where no suggested alternative is needed, whereas the latter is more an *it depends*, and hence often comes with explanations and suggestions. So I think it makes more sense in `retext-equality`? E.g.,

```yml
- type: basic
  note: The word `bugre` is a slur in Portuguese
  considerate:
    - bug report
    - snapshot
  inconsiderate:
    - bugreport
```

In [`race.yml`](https://github.com/retextjs/retext-equality/tree/main/data/en)?

**Comment by wooorm** [comment]
Released in [`retext-equality@5.2.0`](https://github.com/retextjs/retext-equality/releases/tag/5.2.0)!
